index.html

```diff
@@ -113,17 +113,70 @@ <h2>Multi-Post Stories</h2>
               </li>
             </ul>
           </div>
-          <div class="project-button">
-            <a href="projectlink"
-              ><img
-                src="images/Icons/Button/Enable.png"
-                alt="button
-        icon"
-            /></a>
+          <div class="project-button click-project featureProj">
+            <button type="button" class="see-project liveLink">
+              See Project
+            </button>
           </div>
         </div>
       </div>
     </section>
+
+    <!-- Pop Up Window -->
+    <div class="popup hidden">
+      <!-- title and the cancel button-->
+      <div class="popup-heading">
+        <h2>Multi Post Stories</h2>
+        <img
+          class="popup-close"
+          src="images/popup/close.png"
+          alt="close icon"
+        />
+      </div>
```

Comment:
Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, margins)

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/162153933-a94b5d4b-81ad-4721-8247-c3bd10fbe8ca.png) | ![image](https://user-images.githubusercontent.com/10332499/162153497-c8dc324a-6c78-4b29-918e-c0dce2950bff.png)

## Mobile version:
Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/162154517-36032e65-a5c6-4126-82bd-f77c4ad849dc.png) | ![image](https://user-images.githubusercontent.com/10332499/162154367-146172e2-a1bf-4846-b9ee-41ca38a229c7.png)